### ISSUE TITLE
Dots over the image and simple transition

### ISSUE THREAD

**Issue by fmertins** [thiagosf/skitter#41]
@thiagosf how's it going?
Your plugin is awesome and I'm even considering to help with some money :-) Do you accept donations, patreon, etc?

I'd like to ask two questions:

1) Is there any option to show the dots links over the image, instead of below from it? Or, if I'm able to change that by some CSS?

2) I was looking at the full transition list and unfortunately I couldn't find a simple transition like the Bootstrap carousel:
https://getbootstrap.com/docs/4.0/components/carousel/
In other words, I'd like to just pull/push the image from left to right (or right to left) without any aditional effects :-)

**Comment by thiagosf** [comment]
Hello, I'm fine!

Of course, if you want help me to mantain the plugin, you can donate through PayPal, there is a button in side of Skitter's site: https://skitter-slider.net

1. Yep, you are able do that by CSS 
2. I'm sorry, there is not this effect. You can see all available animations like that:
```js
$('.skitter).getAnimations()
```
